config/kubernetes-sigs/org.yaml

```diff
@@ -528,6 +531,18 @@ teams:
     - riverzhang
     - woopstar
     privacy: closed
+  metrics-server-admins:
```

Comment:
Can we instead create a sig-instrumentation directory and move it there?

Reply:
will fix it shortly 👍